lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -49,25 +49,25 @@
        configurations conforms to task arguments.'
     - 'If C(state) is set to C(absent) and virtual machine exists, then the specified virtual machine
       is removed with its associated components.'
-    - 'If C(state) is set to one of the following C(poweredon), C(poweredoff), C(present), C(restarted), C(suspended)
+    - 'If C(state) is set to one of the following C(powered-on), C(powered-off), C(present), C(restarted), C(suspended)
```

Comment:
I would keep all previous states as well in-order to maintain backward compatibility.

Reply:
```suggestion
    - 'If C(state) is set to one of the following C(powered-on), C(poweredon), C(powered-off), C(poweredoff), C(present), C(restarted), C(suspended)
```